index.html

```diff
@@ -0,0 +1,94 @@
+        </div>
+    </header>
+    <main>
+        <div class="main-top">
+            <div class="main-top-text-container">
+                <div class="main-top-text">일상의 모든 물건을  <br>거래해 보세요</div>
+                <a class="main-top-text-button">구경하러 가기</a>
+            </div>
+            
+            <img class="main-top-image" src="./images/onboard/home_top.png" alt="탑이미지">
+        </div>
+
+        <div class="main-section-container">
+            <div class="main-section">
+                <img class="main-section-image" src="./images/onboard/home_01.png" alt="인기아이템템">
+
+                <div class="main-section-text">
+                    <div class="main-section-text-primary">Hot item</div>
+                    <div class="main-section-text-title">인기 상품을 <br class="tablet-br, mobile-br">확인해 보세요</div>
+                    <div class="main-section-text-subtitle">가장 HOT한 중고거래 물품을<br> 판다 마켓에서 확인해 보세요</div>
+
+                </div>
+            </div>
+            
+            <div class="main-section" id="main-section-text-2">
+
+                <div class="main-section-text" >
+                    <div class="main-section-text-primary">Search</div>
+                    <div class="main-section-text-title">구매를 원하는<br class="tablet-br, mobile-br">상품을 검색하세요</div>
+                    <div class="main-section-text-subtitle">구매하고 싶은 물품은 검색해서<br>쉽게 찾아보세요</div>
+                </div>
+                <img class="main-section-image" src="./images/onboard/home_02.png" alt="검색색">
+
+            </div>
+        
+            <div class="main-section">
+                <img class="main-section-image" src="./images/onboard/home_03.png" alt="등록">
+
+                <div class="main-section-text">
+                    <div class="main-section-text-primary">Register</div>
+                    <div class="main-section-text-title">판매를 원하는<br class="tablet-br, mobile-br">상품을 등록하세요</div>
+                    <div class="main-section-text-subtitle">어떤 물건이든 판매하고 싶은 상품을<br>쉽게 등록하세요</div>
+                </div>
+
+            </div>
+        </div>
+        <div class="main-bottom">
+            <div class="main-bottom-text">믿을 수 있는 <br class="tablet-br">판다마켓 중고 거래</div>
+            <img class="main-bottom-image" src="./images/onboard/home_bottom.png" alt="바텀이미지">
+
+        </div>
+
+    </main>
+    <footer>
+        <div class="footer-container">
+            <div class="footer-text">
+                <p>©codeit - 2024</p>
+            </div>
+            <div class="footer-text">
+                <a class="footer-text-link" href="privacy.html">Privacy Policy</a>
+                <a class="footer-text-link" href="terms.html">FAQ</a>
+            </div>
+            <div class="footer-icon">
+                <a href="https://www.facebook.com" target="_blank">            
+                    <img class="footer-icon-item" src="./images/onboard/facebook.png" alt="facebook" link="https://www.facebook.com">
+                </a>

```

Comment:
💊 제안  
이 영역은 로고들의 모음이라고 할 수 있습니다. 관련된 요소들끼리 묶인 목록이니 의미록적인 마크업을 작성한다면 list 관련 태그로 작성하시는 것을 추천드립니다.

```suggestion
            <ul class="footer-icon">
                <li>
                  <a href="https://www.facebook.com" target="_blank">            
                      <img class="footer-icon-item" src="./images/onboard/facebook.png" alt="facebook" link="https://www.facebook.com">
                  </a>
                </li>
```